src/components/countries.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { CountriesContext } from '../contexts/countries';
+import { useSorting, useThemeSwitcher } from '../hooks';
+import { CountriesContextType, Language, TypeCountry } from '../@types/country';
+import { ThemeContext } from '../contexts/theme';
+import { ThemeContextType } from '../@types/theme';
+
+type CountriesProps = {
+    selectedContinent: string | null;
+}
+
+const Countries:React.FC<CountriesProps> = ({ selectedContinent }: CountriesProps) => {
```

Comment:
it's typed in the FC-typing, you don't need to retype! 
```suggestion
const Countries:React.FC<CountriesProps> = ({ selectedContinent }) => {
```